File: GFG/Solution/Medium/43.cs
class Solution {
    public int maxSubarrayXOR(int[] arr, int k) {
        // code here
        int n = arr.Length;
        if( n == 0 || k > n) return 0;
        
        int currentXor = 0;
        for(int i = 0; i < k; i++){
            currentXor ^= arr[i];
        }
        
        int maxXor = currentXor;
        
        for(int i = k; i < n; i++){
            currentXor ^= arr[i] ^ arr[i - k];
            maxXor = Math.Max(maxXor,currentXor);
        }
        
        return maxXor;
    }
}

File: GFG/Solution/Easy/43.cpp
class Solution {
  public:
    int findIndex(string &s) {
        // code here
        int n = s.size();
        int count = 0;
        for(int i = 0; i < n; i++){
            if(s[i] == ')'){
                count++;
            }
        }
        return count;
    }
};